### PR TITLE
Stale user tests

### PR DESCRIPTION
**Pull request by shravanmechineni**
Adding stale user behaviour tests.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/js/welsh_language_test.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ Scenario('@functional @welshLanguage I can set the language with a cookie', asyn
     I.waitForText(Welsh.accessDeniedWelsh, 20, 'h1');
 });
 
-Scenario('@functional @welshLanguage I can set the language with a header', async (I) => {
```
Comment (dfourn) [inline src/test/js/welsh_language_test.js:58]: why remove the `@functional` from here?

Comment (shravanmechineni) [reply src/test/js/welsh_language_test.js:58]: As it's failing on aat and it's a known issue, not fixed yet.

File: src/test/js/account_lockout_reset_password_test.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ Scenario('@functional @unlock My user account is unlocked when I reset my passwo
     I.waitForText('Your password has been changed', 20, 'h1');
     I.see('You can now sign in with your new password.');
     I.saveScreenshot('password-changed.png');
-    I.seeVisualDiff('password-changed.png', {tolerance: 6, prepareBaseImage: false})
+    // I.seeVisualDiff('password-changed.png', {tolerance: 6, prepareBaseImage: false});
```
Comment (radoslaw-orlowski-amido) [inline src/test/js/account_lockout_reset_password_test.js:61]: remove this if not needed?

File: src/test/js/welsh_language_test.js (hunk 1 above)
Comment (radoslaw-orlowski-amido) [reply src/test/js/welsh_language_test.js:58]: Can we keep the old line commented out and with a todo or fixme to change it back once resolved?

Comment (shravanmechineni) [reply src/test/js/welsh_language_test.js:58]: Done.

File: src/test/js/account_lockout_reset_password_test.js (hunk 2 above)
Comment (shravanmechineni) [reply src/test/js/account_lockout_reset_password_test.js]: Done.

**Pull request merged**